<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Main</class>
 <widget class="QMainWindow" name="Form_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiLight</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Изображения/Главная.png</normaloff>Изображения/Главная.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_HomePage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Домашняя страница</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>791</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Cursor">
       <property name="text">
        <string>Введитие название таблицы:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineE_NameTable"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadData">
       <property name="text">
        <string>Загрузить данные</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table_Info">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>791</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>352</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnFormCustomers">
       <property name="text">
        <string>Контрагенты</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Пользователи.png</normaloff>Изображения/Пользователи.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormEmployees">
       <property name="text">
        <string>Сотрудники</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Работники.png</normaloff>Изображения/Работники.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormContracts">
       <property name="text">
        <string>Договоры аренды</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Заказ.png</normaloff>Изображения/Заказ.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormAccounts">
       <property name="text">
        <string>Счета</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Инвентарь.png</normaloff>Изображения/Инвентарь.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormActs">
       <property name="text">
        <string>Акты</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Инвентарь.png</normaloff>Изображения/Инвентарь.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormPayments">
       <property name="text">
        <string>Платежи</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Платеж.png</normaloff>Изображения/Платеж.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormHistory">
       <property name="text">
        <string>История аренд</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Штраф.png</normaloff>Изображения/Штраф.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormInventory">
       <property name="text">
        <string>Инвентарь</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Инвентарь.png</normaloff>Изображения/Инвентарь.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormDecommissionedInventory">
       <property name="text">
        <string>Списанный инвентарь</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Инвентарь.png</normaloff>Изображения/Инвентарь.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFormPrivilegedUsers">
       <property name="text">
        <string>Привилегированные пользователи</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Штраф.png</normaloff>Изображения/Штраф.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_InfoaboutWork">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Перечень таблиц</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>791</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_Query">
       <property name="text">
        <string>Введите SQL-запрос:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textE_Query"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnPerformQuery">
    <property name="geometry">
     <rect>
      <x>999</x>
      <y>180</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить запрос</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>640</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Изображения/Выйти.png</normaloff>Изображения/Выйти.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
